config/checkstyle_non_main_files_checks.xml

```diff
@@ -38,4 +38,19 @@
       <property name="message"
                value="Two or more test cases are required for unit testing in Xpath "/>
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="&lt;!--[^ ]"/>
+        <property name="fileExtensions" value="xml, properties"/>
+        <property name="message" value="space is required in comments"/>
```

Comment:
`space is required in comments` -> space is required after comment start

Reply:
done